tgui/docs/tgui-for-custom-html-popups.md

```diff
@@ -257,3 +257,280 @@ Byond.winget(null, 'url').then((serverUrl) => {
   Byond.command('.quit');
 });
 ```
+# Using TGUI and Byond API for custom HTML popups
```

Comment:
This is doubling what this file already says.

Reply:
Fixed